### PR TITLE
Code Review for [create-animals-table] | Paw Clinic DB Project

### PR DESCRIPTION
**Pull request by luigirazum**
# Hello Dear Code Reviewers Team 👋,
#### - 🎯 General requirements
- 📌 Make sure that you used the correct Gitflow.
  - ✅ The repo was initialized with [curriculum-template-databases](https://github.com/microverseinc/curriculum-template-databases) template, then a `dev` branch created from `main`.
  - ✅ The `dev` branch was set up as default.
  - ✅ `create-animals-table` feature branch was created from `dev`.
- 📌 Make sure that you documented your work in a professional way.
  - ✅ A README file with all the information about the project was created.

#### - 🎯 Project requirements
- 📌 Use [https://github.com/microverseinc/curriculum-template-databases](https://github.com/microverseinc/curriculum-template-databases) template to generate your repo.
  - ✅ `microverseinc/curriculum-template-databases` template was used to initialize the project.
- 📌 Create a database named `vet_clinic`.
  - ✅ image
- 📌 Create a table named `animals` with the following columns: `id`(integer), `name`(string), `date_of_birth`(date), `escape_attempts`(integer), `neutered`(Boolean), `weight_kg`(decimal).
  - ✅ image
  - ✅ image
- 📌 Copy the SQL you used in the previous point into the `schema.sql` file in the generated repository.
  - ✅ The SQL used to create the animals table was added to the `schema.sql` file.
- 📌 Insert the following data:
  - 🚩 Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.
    - ✔ image
  - 🚩 Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.
    - ✔ image
  - 🚩 Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.
    - ✔ image
  - 🚩 Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.
    - ✔ image
- 📌 Copy the SQL you used in the previous point into the `data.sql` file in the generated repository.
  - ✅ The SQL used to add data into `animals` table was added to `data.sql` file.
- 📌 Write queries for the following quests:
  - 🚩 Find all animals whose name ends in "mon".
    - ✔ image
  - 🚩 List the name of all animals born between 2016 and 2019.
    - ✔ image
  - 🚩 List the name of all animals that are neutered and have less than 3 escape attempts.
    - ✔ image
  - 🚩 List the date of birth of all animals named either "Agumon" or "Pikachu".
    - ✔ image
  - 🚩 List name and escape attempts of animals that weigh more than 10.5kg
    - ✔ image
  - 🚩 Find all animals that are neutered.
    - ✔ image
  - 🚩 Find all animals not named Gabumon.
    - ✔ image
  - 🚩 Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
    - ✔ image
- 📌 **Take a screenshot of the results of your queries.**
  - ✅ All previous queries have their screenshot.
- 📌 Copy the SQL you used in the previous point into the `queries.sql` file in the generated repository.
  - ✅ The SQL used to perform the above queries was added to `queries.sql` file.
- 📌 Remember to add the screenshot with the results of your queries to your Pull Request description.
  - ✅ All screenshots results for every query was added into this Pull Request.

### Luis 🙋‍♂️

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg)
+  VALUES ('Agumon', '02/03/2020', 0, true, 10.23);
+
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg)
+  VALUES ('Gabumon', '11/15/2018', 2, true, 8);
+
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg)
+  VALUES ('Pikachu', '06/07/2021', 1, false, 15.04);
```
Comment (Ismailco) [inline data.sql]: - [ ] Kindly make sure to use the correct value of `date_of_birth`, it should be `'2021/01/07'`

File: data.sql [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg)
+  VALUES ('Devimon', '05/12/2017', 5, true, 11);
```
Comment (Ismailco) [inline data.sql]: - [ ] Kindly make sure to use the correct value of `date_of_birth`, it should be `'2017/05/12'`

File: queries.sql [hunk 3]
```diff
@@ -1,3 +1,27 @@
...
+/* - Find all animals that are neutered. */
+SELECT * FROM animals WHERE neutered=true;
+
+/* - Find all animals not named Gabumon. */
+SELECT * FROM animals WHERE name NOT IN ('Gabumon');
+
+/* - Find all animals with a weight between 10.4kg and 17.3kg */
+/*   (including the animals with the weights that equals precisely 10.4kg or 17.3kg) */
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
```
Comment (Ismailco) [inline queries.sql]: - [ ] Kindly make sure to use only one query here because they have the same impact

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @luigirazum, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 3 above)
Comment (luigirazum) [reply queries.sql]: The first query was preserved only:
`SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;`

required change applied with
https://github.com/luigirazum/pawclinic-db/pull/1/commits/937e5eef7aedce4597865abaa9bb423efdbca130

File: data.sql (hunk 1 above)
Comment (luigirazum) [reply data.sql]: Required change applied with
https://github.com/luigirazum/pawclinic-db/pull/1/commits/27e2e12567791b5f6a8dfd16ba2be14416b1892c

File: data.sql (hunk 2 above)
Comment (luigirazum) [reply data.sql]: Required change applied with
https://github.com/luigirazum/pawclinic-db/pull/1/commits/04f37e5c93d43645e16b0f63ce8d996fbdb588a4

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @luigirazum👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**